### ISSUE TITLE
BlSelect - Form Validation

### ISSUE THREAD

**Issue by AykutSarac** [Trendyol/grace#528]
**Is your feature request related to a problem? Please describe.**
Add validation if selected value appears to be `disabled` for form submit

**Describe the solution you'd like**
When submit form it'll check for select input whether the selected value is disabled or not

**Comment by muratcorlu** [comment]
I'm not very clear about the expectation here. As far as I understand, there is a select that has some `disabled` options, but somehow they are also selected. Probably they became disabled later then user's selection. Something like below, right?:

```html
<bl-select name="country" value="tr">
   <bl-select-option value="tr" disabled>Turkey</bl-select-option>
   <bl-select-option value="nl">Netherlands</bl-select-option>
</bl-select>
```

And you expect in this situation select should be in `invalid` state. Do I understand correctly?

**Comment by AykutSarac** [comment]
> I'm not very clear about the expectation here. As far as I understand, there is a select that has some `disabled` options, but somehow they are also selected. Probably they became disabled later then user's selection. Something like below, right?:
> 
> ```
> <bl-select name="country" value="tr">
>    <bl-select-option value="tr" disabled>Turkey</bl-select-option>
>    <bl-select-option value="nl">Netherlands</bl-select-option>
> </bl-select>
> ```
> 
> And you expect in this situation select should be in `invalid` state. Do I understand correctly?

Yes, I expect to see an error when submit if the selected value is no longer available/disabled.

**Comment by muratcorlu** [comment]
That's an interesting topic. I'm not sure what would be the ideal way of handling this situation. I just tried to see how native select act on this. Please check: https://jsfiddle.net/k5x9jyrw/

I set value of the select with a disabled option. Then it automatically unselect it, when you check the value, and it doesn't become invalid. But I'm not sure if this is an ideal approach as well.

@buseselvi User make a selection and saves it. Next they open same form, form comes with the values that user filled before. But selected value of the  user is disabled now. How should we handle this? Do you think it's good to set select invalid in this case, so user will need to make a new selection to be able sae form.
